apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -85,13 +77,10 @@ export function ColumnToggleButton() {
         setOpen(false);
     }, [setOpen]);
```

Comment:
You can safely remove this from the dependency array. You can also do this where applicable in other memo-ized callbacks.

```suggestion
    }, []);
```